dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+FROM node:current-alpine3.11
+EXPOSE 8080/tcp
```

Comment:
after adding environment variable support for config, make sure the app always run on 8080 using the env